src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.ts

```diff
@@ -210,7 +224,7 @@ export class SamHierarchicalAutocompleteComponent {
    * @param searchString 
    */
   private getResults(searchString: string): void {
-    if (this.searchString !== searchString || this.searchString === '') {
+    if (this.searchString !== searchString || (this.searchString === searchString && !this.showResults) || this.searchString === '') {
```

Comment:
Can you split this out into another method call? It gives us more flexibility to change out that logic later without having to touch this code.